ios/FluentUI/Vnext/List/List.swift

```diff
@@ -97,41 +159,102 @@ public struct MSFListView: View {
     }
 }
 
-/// UIKit wrapper that exposes the SwiftUI List implementation
-@objc open class MSFList: NSObject, FluentUIWindowProvider {
+/// Properties that make up section content
+class MSFListSectionStateImpl: NSObject, ObservableObject, Identifiable, MSFListSectionState {
+    func createCell() -> MSFListCellState {
+        return createCell(at: cells.endIndex)
+    }
+
+    func createCell(at index: Int) -> MSFListCellState {
+        guard index <= cells.count && index >= 0 else {
+            preconditionFailure("Index is out of bounds")
+        }
+        let cell = MSFListCellState()
+        cells.insert(cell, at: index)
+        return cell
+    }
+
+    func getCellState(at index: Int) -> MSFListCellState {
+        guard cells.indices.contains(index) else {
+            preconditionFailure("Index is out of bounds")
+        }
+        return cells[index]
+    }
 
-    @objc public init(sections: [MSFListSectionState],
-                      theme: FluentUIStyle? = nil) {
+    func removeCell(at index: Int) {
+        guard cells.indices.contains(index) else {
+            preconditionFailure("Index is out of bounds")
+        }
+        cells.remove(at: index)
+    }
+
+    var id = UUID()
+    @Published var cells: [MSFListCellState] = []
+    @Published var title: String?
+    @Published var backgroundColor: UIColor?
+    @Published var hasDividers: Bool = false
+
+    var count: Int {
+        return cells.count
+    }
+
+    var style: MSFHeaderFooterStyle {
+        get {
+            return headerTokens.style
+        }
+        set {
+            headerTokens.style = newValue
+            headerTokens.updateForCurrentTheme()
```

Comment:
The call to updateForCurrentTheme() should be in the didSet handler of the tokens style property.

Examples:
- https://github.com/microsoft/fluentui-apple/blob/b2f6b63cbfd77d6e8bde02e3d9ead05f8d08227d/ios/FluentUI/Avatar/Avatar.swift#L486
- https://github.com/microsoft/fluentui-apple/blob/b2f6b63cbfd77d6e8bde02e3d9ead05f8d08227d/ios/FluentUI/Avatar/AvatarTokens.swift#L68